Cargo.toml

```
@@ -22,6 +22,7 @@ clickhouse = ["http_wait"]
 cncf_distribution = []
 consul = []
 dynamodb = []
+databend = []
```

It's required to enable http strategy for this module explicitly

```suggestion
databend = ["http_wait"]
```

---

This one is still relevant. Also please apply rustfmt and LGTM!

Thank you!